docs/data-collection/notes-scanning.md

```diff
@@ -162,6 +162,8 @@ Adjust settings by pressing the respective button and then turning the central k
 
 !!! warning "The central knob (button 1) will turn off the alarm if pushed when the alarm is on"
 
+!!! warning "On {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}, the alarm can be turned off by touching the tactil screen on the bore of the scanner"
```

Comment:
Does the central knob work for this too? or is it just different?

Reply:
There is no central knob on the Vida, the alarm can only be turned off by touching the screen